examples/http-swagger/src/example/server.clj

```diff
@@ -42,11 +43,25 @@
       [["/swagger.json"
         {:get {:no-doc true
                :swagger {:info {:title "my-api"
-                                :description "with reitit-http"}}
+                                :description "with reitit-http"}
+                         ;; used in /secure APIs below
+                         :securityDefinitions {"auth" {:type :apiKey
+                                                       :in :header
+                                                       :name "Example-Api-Key"}}}
                :handler (swagger/create-swagger-handler)}}]
+       ["/openapi.json"
```

Comment:
we could change the default into swagger-ui handler to read the `openapi.json` over `swagger.json` if that exists?